packages/components/components/elvis-app-bridge/src/react/getLinks.ts

```diff
@@ -0,0 +1,45 @@
+export interface SystemLink {
+  url: string;
+  name: string;
+  iconInfo: {
+    rotation: number;
+    iconLetters: string;
+  };
+}
+
+export const getLinks = (targetId: string): SystemLink[] => {
+  return [
+    {
+      name: 'DROPS',
+      url: `https://drops.elvia.io/meteringpoint/${targetId}`,
+      iconInfo: {
+        rotation: 67.5,
+        iconLetters: 'Dr',
+      },
+    },
+    {
+      name: 'MSIm',
+      url: `https://msim.elvia.io/tickets/meteringpoint/${targetId}`,
+      iconInfo: {
+        rotation: -180,
+        iconLetters: 'Ms',
+      },
+    },
+    {
+      name: 'MDMx',
+      url: `https://mdmx.elvia.io/meteringpoint?id=${targetId}`,
```

Comment:
Hm. Thill always goes to the prod environment. I think we will get a feature request soon asking if the App Bride can respect the current environment 🥲. I think this is good enough 🤷🏽